tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -1257,3 +1257,28 @@ def test_external_new_user_login_and_check_count(session):
         finally:
             update_rhsso_settings_in_satellite(revert=True)
             delete_rhsso_user(user_details['username'])
+
+
+@tier2
+def test_positive_test_connection_functionality(session, ldap_data, ipa_data):
+    """Verify for a positive test connection response
+
+    :id: 5daf3976-9b5c-11ea-96f8-4ceb42ab8dbc
+
+    :steps: Assert test connection of AD and IPA.
+
+    :expectedresults: Positive test connection of AD and IPA
+    """
+    with session:
+        assert (
+            session.ldapauthentication.testConnection(
+                {'ldap_server.host': ldap_data['ldap_hostname']}
+            )
+            is None
+        )
```

Comment:
```suggestion
for ldap_host in (ldap_data['hostname'], ipa_data['hostname']):
       assert not session.ldapauthentication.test_connection({'ldap_server.host':ldap_host})
```
everything in two lines, let me if that works for you ?

Reply:
Even I thought of using 'for' but somehow I wasn't able to run it and I don't remember what I wrote. That's why i switched to the other way.
Thanks, this works.
**Changes made**.